app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -511,4 +513,23 @@ public static void disableRateLimitPopup(boolean disable) {
                 .putBoolean(DISABLE_RATE_LIMIT_POPUP, disable)
                 .apply();
     }
+
+    public static void updateLazyMediaDownloadResult() {
+        String prefKey = String.format(LAZY_MEDIA_DOWNLOAD_COMPLETE, AppUtils.getCurrentAppVersion());
```

Comment:
I think it's dangerous that this is implicit.

I think we we should either receive this argument as a parameter, or we should explicitly state in method name what the scope it returns over (IE "ForCurrentlySeatedApp")